src/Services/addressService.ts

```diff
@@ -0,0 +1,59 @@
+import { addressRepo } from '../Repositories/addressRepository';
+import { Address } from '../Entities/Address';
+import { AddressRequest2DTO, AddressRequestDTO, AddressResponseDTO } from '../DTO/addressDTO';
+
+export const createAddress = async (addressRequestDTO: AddressRequest2DTO) => {
+    try {
+        const save = await addressRepo.save(addressRequestDTO);
+        return convertToDTO(save);
+    } catch (err) {
+        return err.message === 'Something went wrong!' ? { err: err.message } : { err: 'Something went wrong!- we are working on it!' };
+    }
+};
+
+export const getAddresses = async () => {
+    const addresses = await addressRepo.find();
+    const addressDTOs: AddressResponseDTO[] = addresses.map(address => convertToDTO(address));
+    return addressDTOs;
+};
+
+export const getAddressById = async (id: number) => {
+    const response = await addressRepo.findOneById(id);
```

Comment:
You are using a deprecated version of findOneById, change the name to use your method from addressRepo
```suggestion
    const response = await addressRepo.findOneByID(id);
```